mls-rs/src/client.rs

```diff
@@ -410,12 +411,13 @@ where
     }
 
     #[cfg_attr(all(feature = "ffi", not(test)), safer_ffi_gen::safer_ffi_gen_ignore)]
-    pub fn to_builder(&self) -> ClientBuilder<MakeConfig<C>> {
+    pub fn to_builder(&self, maybe_now_time: Option<MlsTime>) -> ClientBuilder<MakeConfig<C>> {
```

Comment:
I would suggest aligning with the existing code and call this `timestamp` consistently throughout:

```suggestion
    pub fn to_builder(&self, timestamp: Option<MlsTime>) -> ClientBuilder<MakeConfig<C>> {
```

Reply:
Makes sense; done.